examples/wallet_contract_caller_script/src/main.sw

```diff
@@ -7,10 +7,16 @@ fn main() {
     let contract_address = 0x9299da6c73e6dc03eeabcce242bb347de3f5f56cd1c70926d76526d7ed199b8b;
     let caller = abi(Wallet, contract_address);
     let amount_to_send = 200;
-    let recipient_address = Address::from(0x9299da6c73e6dc03eeabcce242bb347de3f5f56cd1c70926d76526d7ed199b8b);
-    caller.send_funds {
```

Comment:
I feel like the original looks easier to read. What would cargo fmt produce here?

Reply:
@JoshuaBatty Perhaps I should tweak the defaults to allow longer lines? Or perhaps I can aim something like

```rust
    caller
        .send_funds {
            gas: 10000,
            coins: 0,
            asset_id: ZERO_B256,
        }(amount_to_send, recipient_address);
        ````

Reply:
Yeah I think that sounds good. that looks a lot nice IMO

Reply:
[Any better now](https://github.com/FuelLabs/sway/pull/5207/files#diff-08ef0bbba0955879505c2ae86a078223951e9fafa935c6af77679cf07609cd92R13-R18) @JoshuaBatty ?

Reply:
Yeah that's better, I still think the first part should be on the same line though no?

ie 

```sway
caller.send_funds {
   ....
````   

Here is some unrelated formatted code in rust 

```rust
self.source_to_path_map
    .read()
    .unwrap()
    .get(source_id)
    .unwrap()
    .clone()
```

Reply:
This is news to me, but in `cargo fmt` the second element in the chain will be inline if the first element is shorter than 5  letters.


```rust
   fuel.really_long_field
        .other_really_long_field
        .foo()
        .bar()
        .baz
        .quux()
        .yet_another_call()
        .to_go_above_max_line_length();
```

vs 

```rust
    fuelcore
        .really_long_field
        .other_really_long_field
        .foo()
        .bar()
        .baz
        .quux()
        .yet_another_call()
        .to_go_above_max_line_length();
  ```

I'll make this rule happen

Reply:
Does it look better now @JoshuaBatty ?

Reply:
agh ok didn't realise the rule was 5 chars. yep looks good thanks.